src/NewSessionView.elm

```diff
@@ -187,65 +185,60 @@ view context model =
                                     [ text (DateUtils.displayDateWithoutTime d) ]
                             )
             in
-                div [ class "form-group" ]
-                    [ div [ class "input-group", onInput UpdateNewSessionDate ]
-                        [ label [ for "day-input" ] [ text "Date " ]
-                        , br [] []
-                        , select [ id "day-input" ]
+                div []
```

Comment:
So many anonymous `div`s 😢  😢  😢  Please please try and reduce these, it undermines the descriptive nature of BEM.